Library/OcDeviceMiscLib/SetResizableBar.c

```diff
@@ -102,7 +174,7 @@ SetResizableBarOnDevice (
 
   ChangedBars = FALSE;
```

Comment:
We also have the `ChangedBars` logic here, in the original code.

Reply:
Sorry, what does this refer to?

Reply:
You asked @xCuri0 to remove this logic in his (modified) clone of this method. I think it should be removed here too?

Reply:
(I'm picking up on this because @xCuri0 said that he got the code which he was asked to change from the original code!)

Reply:
Despite the other discussion below, I would still feel the same about this: if it was worth changing in the modified clone of this method, then surely it's worth changing here, in the original method in the same file, no?

Reply:
As I said for another change, feel free to, but I wouldn't require this for merging.

Reply:
PS If you were agreeing that it's theoretically worth doing that more than it's fair to ask @xCuri0 to pick up, for his PR to be accepted, then apologies for misunderstanding understanding that point - and - although I would still think they really are minor, and related, changes - of course be happy to do a small subsequent PR to make the additional changes (no new static asserts!) - that I am suggesting.